projects/distributed-tracing/src/shared/dashboard/data/graphql/graphql-options-data-source.model.ts

```diff
@@ -0,0 +1,10 @@
+import { TableControlOption } from '@hypertrace/components';
+import { Observable } from 'rxjs';
+import { GraphQlDataSourceModel } from './graphql-data-source.model';
+
+export abstract class GraphqlOptionsDataSourceModel<
```

Comment:
Is this old? I think this got renamed in a previous PR

Reply:
Good catch.